esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -1,20 +1,35 @@
 #!/bin/bash
 
 # ESP site creation script
-# Michael Price, December 2010
+# See /docs/dev/deploying.rst for documentation on how to deploy.
+
+set -e -o pipefail
```

Comment:
Actually, it looks like you can't use the ``-u`` flag, because of the ``DIFF`` variable. Though if you wanted to, you could make sure to set ``DIFF`` in all scenarios, and then you can use the ``-u`` flag.